nixos/doc/manual/installation/installing.xml

```diff
@@ -70,9 +70,12 @@
 
    <para>
     If you would like to continue the installation from a different machine you
-    need to activate the SSH daemon via <command>systemctl start
-    sshd</command>. You then must set a password for either <literal>root</literal> or
-    <literal>nixos</literal> with <command>passwd</command> to be able to login.
+    can use activated SSH daemon. You need to copy your ssh key to either
+    <literal>/home/nixos/.ssh/authorized_keys</literal> or
+    <literal>/root/.ssh/authorized_keys</literal> (Tipp: For headless machines, this
+    can be done by modifying the installer).  Alternativly you must set a password
+    for either <literal>root</literal> or <literal>nixos</literal> with
+    <command>passwd</command> to be able to login.
    </para>
```

Comment:
Note that this may be misleading. The USB image is squashfs'd so it cannot be modified offline. I'm not sure how to rephrase this to better explain this.

Though it might not need a rephrase. Its current ambiguity can and should be interpreted in a way where the end-user can modify the installer [and generate a new one using the nix expressions].

Reply:
Improved.